src/BugsnagUnity/Payload/Event.cs

```diff
@@ -34,6 +33,8 @@ internal Event(string context, Metadata metadata, App app, Device device, User u
                     session.Events.IncrementUnhandledCount();
                 }
             }
+            this.AddToPayload("session", session);
```

Comment:
`IncrementHandledCount` and `IncrementUnhandledCount` don't look like they will increment the handled/unhandled count for the Android/Cocoa layer - will that cause issues?

Reply:
After the event is sent the Client calls UpdateSession in the NativeClient which updates the handled/unhandled values in the native layers.